### PR TITLE
Upgrade/fixed header

### PR DESCRIPTION
**Pull request by mmumby**
- Moved all the header logic into a common component to have consistent styling - more control over the header css
- Fixes some alignment issues
- Moved the update status outside of the approvedCode If block so that the status will be updated
- Updated logic surrounding displaying the "Client delay reason" or "rejected status reason"

### CODE REVIEW

File: services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js [hunk 1]
```diff
@@ -543,82 +542,75 @@ export class NoticeOfWorkApplication extends Component {
...
+                </Button>
+              </Dropdown>
+            </>
+          )
+        }
+        tabEditActions={
+          <div className="center">
+            <Popconfirm
+              placement="bottomRight"
+              title="You have unsaved changes, Are you sure you want to cancel?"
```
Comment (exzizt) [inline services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js:575]: Are --> are

File: services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js [hunk 2]
```diff
@@ -216,100 +214,67 @@ export class NOWPermitGeneration extends Component {
...
-          <NOWProgressStatus tab="DFT" />
-        </div>
+          tabName={`Draft ${nowType}`}
+          handleDraftPermit={this.handleDraftPermit}
+          fixedTop={this.props.fixedTop}
+          tabEditActions={
+            <>
+              <Popconfirm
+                placement="bottomRight"
+                title="You have unsaved changes, Are you sure you want to cancel?"
```
Comment (exzizt) [inline services/core-web/src/components/noticeOfWork/applications/permitGeneration/NOWPermitGeneration.js:262]: Are --> are

All occurrences

File: services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js [hunk 3]
```diff
@@ -259,9 +258,8 @@ export class ProcessPermit extends Component {
   handleApplication = (values, code) => {
     if (code === approvedCode) {
       return this.handleApprovedApplication(values);
-    } 
-      return this.afterSuccess(values, "This application has been successfully rejected.", code);
-    
+    }
+    return this.afterSuccess(values, "This application has been successfully rejected.", code);
```
Comment (exzizt) [inline services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js:262]: "successfully rejected" awesome :P

File: services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js (hunk 1 above)
Comment (mmumby) [reply services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js]: fixed

**Review by exzizt — APPROVED** [review]

**Pull request merged**